src/lib/containers/evaluation_queues/EvaluationEditor.tsx

```diff
@@ -0,0 +1,227 @@
+import { CreatedOnByUserDiv } from './CreatedOnByUserDiv'
+
+const defaultEvaluationStatus = EvaluationStatus.PLANNED
+
+export type EvaluationEditorProps = {
+  /** session token to make authenticated API calls */
+  readonly sessionToken: string
+  /** Use if UPDATING an existing Evaluation. Id of the evaluation to edit */
+  readonly evaluationId?: string
+  /** Use if CREATING a new Evaluation. Id of the Entity that will be associated with the Evaluation */
+  readonly entityId?: string
+  /** If true, the "Created on" date will be displayed in UTC time */
+  readonly utc: boolean
+}
+
+/**
+ * Edits basic properties of an Evaluation
+ */
+export const EvaluationEditor: React.FunctionComponent<EvaluationEditorProps> = ({
+  sessionToken,
+  evaluationId,
+  entityId,
+  utc,
+}: EvaluationEditorProps) => {
+  if (evaluationId && entityId) {
+    throw new Error('please use either evaluationId or entityId but not both')
+  }
+
+  const [error, setError] = useState<SynapseClientError>()
+
+  const [name, setName] = useState<string>('')
+  const [description, setDescription] = useState<string>('')
+  const [
+    submissionInstructionsMessage,
+    setSubmissionInstructionsMessage,
+  ] = useState<string>('')
+  const [submissionReceiptMessage, setSubmissionReceiptMessage] = useState<
+    string
+  >('')
+  const [status, setStatus] = useState<EvaluationStatus>(
+    defaultEvaluationStatus,
+  )
+
+  const [evaluation, setEvaluation] = useState<Evaluation>({
+    contentSource: entityId,
+  })
+
+  useEffect(() => {
+    setName(evaluation.name ?? '')
+    setDescription(evaluation.description ?? '')
+    setSubmissionInstructionsMessage(
+      evaluation.submissionInstructionsMessage ?? '',
+    )
+    setSubmissionReceiptMessage(evaluation.submissionReceiptMessage ?? '')
+    setStatus(evaluation.status ?? defaultEvaluationStatus)
+  }, [evaluation])
+
+  useEffect(() => {
+    // if we initially passed in a update the we retrieve a new Evaluation
+    if (evaluationId) {
+      //clear error
+      setError(undefined)
+      getEvaluation(evaluationId, sessionToken)
+        .then(retrievedEvaluation => {
+          setEvaluation(retrievedEvaluation)
+        })
+        .catch(error => setError(error))
+    }
+  }, [evaluationId, sessionToken])
+
+  const onSave = () => {
+    // clear out error
+    setError(undefined)
+    const newOrUpdatedEvaluation: Evaluation = {
+      ...evaluation,
+      name,
+      description,
+      submissionInstructionsMessage,
+      submissionReceiptMessage,
+      status,
+    }
+
+    const promise = newOrUpdatedEvaluation.id
+      ? updateEvaluation(newOrUpdatedEvaluation, sessionToken)
+      : createEvaluation(newOrUpdatedEvaluation, sessionToken)
+
+    promise
+      .then(evaluation => setEvaluation(evaluation))
+      .catch(error => setError(error))
+  }
+
+  // create delete callback if the evaluation has id
+  const onDelete = evaluation?.id
```

Comment:
we probably want delete success callback so we can take the user back to the previous page once it is deleted.